packages/explorer/bin/explorer.ts

```diff
@@ -13,7 +13,7 @@ const __dirname = path.dirname(__filename);
 const argv = minimist(process.argv.slice(2));
 const port = argv.port || process.env.PORT || 13690;
 const chainId = argv.chainId || process.env.CHAIN_ID || 31337;
-const env = argv.env || process.env.NODE_ENV || "production";
+const env = argv.dev ? "development" : "production";
```

Comment:
what about just
```suggestion
const isDev = argv.dev;
```

then below

```ts
if (isDev) {
  ...
} else {
  ...
}
```
(rather not conflate the notion of "env")